docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md

```diff
@@ -1,7 +1,20 @@
 ---
 title: "E-commerce"
 sidebar_position: 105
+hide_title: true
 ---
+# E-commerce Quickstart
```

Comment:
Why? If you want different sidebar label and title, you can use:
```
title: "E-commerce Quickstart"
sidebar_label: "E-commerce"
```

Reply:
TIL